sdk/mysql/Azure.ResourceManager.MySql/tests/Scenario/MySqlFlexibleServerTests.cs

```diff
@@ -17,6 +18,8 @@ public class MySqlFlexibleServerTests: MySqlManagementTestBase
         public MySqlFlexibleServerTests(bool isAsync)
             : base(isAsync)
         {
+            BodyKeySanitizers.Add(new BodyKeySanitizer("https://fakeaccout.blob.windows.core.net/fakecontainer") { JsonPath = "properties.importSourceProperties.storageUrl" });
+            BodyKeySanitizers.Add(new BodyKeySanitizer("fakesastoken") { JsonPath = "properties.importSourceProperties.sasToken" });
```

Comment:
Consider to abstract "fakesastoken" to a const string field, so it can be used for this and in the test.

Reply:
updated in new commit